perses/app/setup_relative_calculation.py

```diff
@@ -517,26 +519,22 @@ def run_setup(setup_options):
-                                                                                             reassign_velocities=False,
-                                                                                             n_restart_attempts=20,
-                                                                                             splitting="V R O R V",
-                                                                                             constraint_tolerance=1e-06),
+                hss[phase] = HybridSAMSSampler(mcmc_moves=mcmc.LangevinDynamicsMove(timestep=timestep,
+                                                                                    collision_rate=5.0 / unit.picosecond,
+                                                                                    n_steps=n_steps_per_move_application,
+                                                                                    reassign_velocities=False,
+                                                                                    n_restart_attempts=20),
                                                hybrid_factory=htf[phase], online_analysis_interval=setup_options['offline-freq'],
                                                online_analysis_minimum_iterations=10,flatness_criteria=setup_options['flatness-criteria'],
                                                gamma0=setup_options['gamma0'])
                 hss[phase].setup(n_states=n_states, n_replicas=n_replicas, temperature=temperature,storage_file=reporter,lambda_protocol=lambda_protocol,endstates=endstates)
             elif setup_options['fe_type'] == 'repex':
-                hss[phase] = HybridRepexSampler(mcmc_moves=mcmc.LangevinSplittingDynamicsMove(timestep=timestep,
-                                                                                             collision_rate=5.0 / unit.picosecond,
-                                                                                             n_steps=n_steps_per_move_application,
-                                                                                             reassign_velocities=False,
-                                                                                             n_restart_attempts=20,
-                                                                                             splitting="V R O R V",
-                                                                                             constraint_tolerance=1e-06),
-                                                                                             hybrid_factory=htf[phase],online_analysis_interval=setup_options['offline-freq'])
+                hss[phase] = HybridRepexSampler(mcmc_moves=mcmc.LangevinDynamicsMove(timestep=timestep,
+                                                                                     collision_rate=5.0 / unit.picosecond,
```

Comment:
I think we want to use 1/ps for the collision rate unless we have a compelling reason to use a higher rate.